nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/PutDatabaseRecord.java

```diff
@@ -146,6 +148,18 @@
 
     protected static Set<Relationship> relationships;
 
+    public static final Validator TRANSLATE_FIELD_EXPRESSION_VALIDATOR = new Validator() {
```

Comment:
All the same comments for ConvertJSONToSQL apply here as well

Reply:
Copy that.Sorry reply so late because I just ended my long vacation, I would modify the code as soon as possible